racingcar-java/src/main/java/step2/StringCalculator.java

```diff
@@ -0,0 +1,60 @@
+package step2;
+
+public final class StringCalculator extends ArithmeticCalculator {
+
+    private final static String[] operators = {"+", "-", "*", "/"};
+
+    public int calculation(final String str) {
+        String[] input =  this.validation(str);
```

Comment:
```suggestion
        String[] input =  validation(str);
```
자기자신의 메소드를 호출한다면 굳이 this라는 키워드가 불필요해보이네요 :)

Reply:
this 키워드를 추가할지 말지 고민을 많이 했었는데 않넣는게 좋겠근요 .. ! 의견 감사합니다 👍

Reply:
this 는 동일한 이름을 가진 요소가 들어올 때 사용하는 것이 일반적입니다.  
주로 생성자나, 정적 팩토리 메서드에서 사용하죠 :)